app/src/main/res/values/strings.xml

```diff
@@ -32,4 +32,8 @@
     <string name="women">Women</string>
     <string name="men">Men</string>
     <string name="kids">Kids</string>
+
+    <string name="categories">Categories</string>
```

Comment:
Please use more specific names for all added strings. For example `label_categories`, `hint_categories` or whatever else that would be more suitable in your case. Resources should be name in manner `what_is_it_specific_key`: `title_home`, `hint_password`, `error_wrong_password` and so on

Reply:
renamed